core/runtime/src/main/java/org/eclipse/sw360/antenna/workflow/AntennaWorkflow.java

```diff
@@ -75,6 +81,17 @@ public AntennaWorkflow(AntennaWorkflowConfiguration antennaWFConfig) {
         }
     }
 
+    private String makeStringForFailCausingResults(Map<String, String> failCausingResults) {
```

Comment:
A String might not be the best option to transport information which might later get rendered in reports or get reported.